specs/account_spec.rb

```diff
@@ -134,38 +130,80 @@
       }.must_raise ArgumentError
     end
   end
+# Added a block to test add_owner method:
+  describe "Account#add_owner" do
+    it "Must be created, account owner must be type of Bank::Owner" do
+      account = Bank::Account.new(2323, 300)
+      account.add_owner(12, "Kuleniuk")
+      account.owner.must_be_kind_of Bank::Owner
+      account.owner.wont_be_nil
+    end
+  end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Wave 2" do
+describe "Wave 2" do
   describe "Account.all" do
-    it "Returns an array of all accounts" do
-      # TODO: Your test code here!
-      # Useful checks might include:
-      #   - Account.all returns an array
-      #   - Everything in the array is an Account
-      #   - The number of accounts is correct
-      #   - The ID and balance of the first and last
-      #       accounts match what's in the CSV file
-      # Feel free to split this into multiple tests if needed
+    it "Account.all returns an array" do
+      Bank::Account.all.must_be_kind_of Array
+    end
+    it "Everything in the array is an Account" do
+      all_accounts = Bank::Account.all
+      all_accounts.each do |account|
+        account.must_be_kind_of Bank::Account
+      end
+    end
+    it " The number of accounts is correct" do
+      all_accounts = Bank::Account.all
+      all_accounts.length.must_be :==, 12
+    end
+    it " - The ID and balance of the first and last
+          accounts match what's in the CSV file" do
+      csv = CSV.read("support/accounts.csv", 'r')
+      expected_id_first = csv[0][0].to_i
+      expected_balance_first = csv[0][1].to_i/100.0
+      expected_id_last = csv[11][0].to_i
+      expected_balance_last = csv[11][1].to_i/100.0
+      all_accounts = Bank::Account.all
+      all_accounts[0].id.must_equal expected_id_first
+      all_accounts[0].balance.must_equal expected_balance_first
+      all_accounts[11].id.must_equal expected_id_last
+      all_accounts[11].balance.must_equal expected_balance_last
     end
   end
 
   describe "Account.find" do
     it "Returns an account that exists" do
-      # TODO: Your test code here!
+      result = Bank::Account.find(1213)
+      result.must_be_kind_of Bank::Account
+      # Can't figure out why next line does not work:
+      # I assume it is connected with the fact that
+      # object references are different in result and
+      # in all_accounts
+      # all_accounts = Bank::Account.all
+      # all_accounts.must_include result
     end
 
     it "Can find the first account from the CSV" do
-      # TODO: Your test code here!
+      csv = CSV.read("support/accounts.csv", 'r')
+      result = Bank::Account.find(csv[0][0].to_i)
+      result.id.must_be :==, Bank::Account.all[0].id
```

Comment:
This works fine, but you should be aware that Minitest has a `.must_equal` that functions the same as `.must_be :==`.